src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -871,6 +871,62 @@ type TaskSeq =
+    /// </summary>
+    ///
+    /// <param name="predicate">A function that evaluates to false when no more items should be skipped.</param>
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
+    static member skipWhile: predicate: ('T -> bool) -> source: TaskSeq<'T> -> TaskSeq<'T>
+
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips elements of the underlying sequence while the
+    /// given asynchronous function <paramref name="predicate" /> returns <see cref="true" />, and then yields the
+    /// remaining elements. The first element where the predicate returns <see cref="false" /> is returned, which
+    /// means that this function will skip 0 or more elements (see also <see cref="TaskSeq.skipWhileInclusive" />).
+    /// If <paramref name="predicate" /> is synchronous, consider using <see cref="TaskSeq.skipWhile" />.
+    /// </summary>
+    ///
+    /// <param name="predicate">An asynchronous function that evaluates to false when no more items should be skipped.</param>
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
+    static member skipWhileAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
+
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips elements of the underlying sequence until the given
+    /// function <paramref name="predicate" /> returns <see cref="false" />, also skips that element
+    /// and then yields the remaining elements (see also <see cref="TaskSeq.skipWhile" />). This function skips
+    /// at least one element of a non-empty sequence, or returns the empty task sequence if the input is empty.
+    /// If <paramref name="predicate" /> is asynchronous, consider using <see cref="TaskSeq.skipWhileInclusiveAsync" />.
+    /// </summary>`
+    ///
+    /// <param name="predicate">A function that evaluates to false when no more items should be skipped.</param>
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
+    static member skipWhileInclusive: predicate: ('T -> bool) -> source: TaskSeq<'T> -> TaskSeq<'T>
+
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips elements of the underlying sequence until the given
+    /// function <paramref name="predicate" /> returns <see cref="false" />, also skips that element
+    /// and then yields the remaining elements (see also <see cref="TaskSeq.skipWhileAsync" />). This function skips
```

Comment:
```suggestion
    /// and then yields the remaining elements (see also <see cref="TaskSeq.skipWhileInclusiveAsync" />). This function skips
```

Reply:
I think this one is currently correct, actually (we should not `see` to the function currently hovered over):
![image](https://github.com/fsprojects/FSharp.Control.TaskSeq/assets/16015770/5f1772bd-8f3a-41ff-94db-857ba2e10ccf)